exec/executil.go

```diff
@@ -232,7 +232,7 @@ func RunPS(cmd string) (string, error) {
 	defer out.Close()
 
 	if err := cmdExec.Start(); err != nil {
-		return "", errors.New("start powershell.exe process error")
+		return "", errkit.New("start powershell.exe process error")
```

Comment:
I guess that if the intention was to propagate the error, we need also the original one here?

Also shall we handle the pipes creation errors as well, which are now ignored?

Reply:
propogated error 👍 

do you have any reference for handling pipes creation errors , we were handling exitError because exec package wraps exit errors with exec.ExitError along with process info and stuff , i couldn't find any wrappers for pipe errors, did you perhaps mean by checking in error message ?

Reply:
I was referring to this code block:
```go
in, _ := cmdExec.StdinPipe()
errorOut, _ := cmdExec.StderrPipe()
out, _ := cmdExec.StdoutPipe()
```
now the second return value/error is ignored. I was thinking if maybe it was worth checking it and returning/wrapping if we are unable to create the pipe

Reply:
done